core/framework/config.py

```
@@ -0,0 +1,168 @@
+        if not file_path.exists():
+            return {}
+        
+        with open(file_path, 'r') as f:
+            return yaml.safe_load(f) or {}
+    
+    @staticmethod
+    def _load_base() -> Dict[str, Any]:
+        """Load base configuration."""
+        base_path = Path(__file__).parent.parent / "configs" / "base.yaml"
+        return ConfigManager._load_yaml(base_path)
+    
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)